pkg/background/mutate/mutate.go

```diff
@@ -69,7 +70,7 @@ func (c *MutateExistingController) ProcessUR(ur *kyvernov1beta1.UpdateRequest) e
 		return err
 	}
 
-	for _, rule := range policy.GetSpec().Rules {
+	for _, rule := range autogen.ComputeRules(policy) {
```

Comment:
Do we want to enable auto-gen for mutate existing policies? As the target resources for such policies are not defined in `match.kinds` but `spec.mutate.targets`.

Reply:
I think we want to have the same behavior, autogen rules are present when mutating the policy so...

Reply:
MutateExisting policies mutate existing resources while the triggers are defined in the `match`. Not sure if auto-gen applies to this type of policy, same thing for generate policies.

Can we remove this for now? If there are use cases we can design and add the support.